examples/html-css/index.html

```diff
@@ -3,10 +3,15 @@
   <head>
     <meta charset="utf-8" />
     <title>Design System</title>
-    <link rel="stylesheet" href="../../packages/core/dist/index.css" />
+    <link rel="stylesheet" href="../../packages/design-system/dist/css/index.css" />
```

Comment:
What happened to using unpkg?
https://unpkg.com/

Reply:
Usage of https://unpkg.com/ didn't work, it cannot find the font & images folder. I'm thinking that could be because current npm release has the variables for font/images at `./` folder. After the release of this PR which changes the variables to look for fonts/images at `../` folder, I'll re-test the use of https://unpkg.com/

Reply:
Have updated to use https://unpkg.com/